app/assets/stylesheets/nfg_ui/network_for_good/admin/color_variations/nfg_theme/custom/_activity_feed.scss

```diff
@@ -0,0 +1,7 @@
+@each $color, $value in $colors {
+  .activity-feed-item {
+    &.#{nth($color, 1)} {
+      .activity-icon { background-color: $value; }
+    }
+  }
+}
```

Comment:
@kylejmorin this is a note that we should probably go ahead and code up the activity feed components in ruby :) particularly since they are there (just not given any life yet) waiting for us ;)